src/main/java/rat/tasks/RatTaskManager.java

```diff
@@ -47,10 +48,10 @@ protected void getDataFromFile() {
                 this.taskList.add(new ToDo(taskData[2]));
```

Comment:
Will having a taskData class instead of hardcoding the indices to handle the conversion better for future format changes? It might not be necessary but a good improvement to have.